src/interaction-handlers/ticket/categorizeCrash.ts

```diff
@@ -26,15 +26,16 @@ export class ButtonHandler extends InteractionHandler {
 
 		switch (crashType) {
 			case 'CP':
-				await interaction.channel.send(`**Click the copy button and paste it here.**\n${plsBePatientTY}`);
+				await interaction.channel.send(`**Click the copy symbol (next to the symbol with an arrow pointing directly up) and paste it here.**\n${plsBePatientTY}`);
```

Comment:
This text is a little wordy and confusing. Would this maybe make more sense?

```suggestion
				await interaction.channel.send(`**Click the copy icon on the right side and paste the results here.**\n${plsBePatientTY}`);
```

Reply:
yeah that would, I'll commit that